src/services/dbtTestService.ts

```diff
@@ -103,10 +103,18 @@ export class DbtTestService {
       return;
     }
 
-    const patchPath = node?.patch_path.includes("://")
+    const patchPath = node?.patch_path?.includes("://")
       ? path.join(project.projectRoot.fsPath, node.patch_path.split("://")[1])
       : node.patch_path;
 
+    if (!patchPath) {
```

Comment:
This seems unrelated to this change. The patch_path will always be there, if there is no file exists check it should always exist.

Reply:
and this is the file exist check